Product

```diff
@@ -0,0 +1,34 @@
+class Product: 
+    def __init__(self, sale_price, number_products, purchase_price, name, brand, image, barcode, instructions, 
+        size):
+        self.sale_price = sale_price # needs to be an integer
```

Comment:
Pep8 convention says that when you write an inline comment you must use 2 spaces before the comment.

Reply:
When updating my program, I was aware of this suggestion. Hence, you can now see that I applied the Pep8 convention.